cardano-node/src/Cardano/Node/Configuration/POM.hs

```diff
@@ -704,8 +705,7 @@ defaultPartialNodeConfiguration =
     , pncConsensusMode = Last (Just Ouroboros.defaultConsensusMode)
       -- https://ouroboros-network.cardano.intersectmbo.org/ouroboros-network/Ouroboros-Network-Diffusion-Configuration.html#v:defaultConsensusMode
     , pncEnableP2P     = Last (Just EnabledP2PMode)
-    , pncPeerSharing   = Last (Just Ouroboros.defaultPeerSharing)
-      -- https://ouroboros-network.cardano.intersectmbo.org/ouroboros-network/Ouroboros-Network-Diffusion-Configuration.html#v:defaultPeerSharing 
+    , pncPeerSharing   = mempty
```

Comment:
There should have a comment here pointing out the default is actually defined in `makeNodeConfiguration`.